tox.ini

```diff
@@ -4,7 +4,7 @@ skip_missing_interpreters = True
 
 [gh-actions]
 python =
-  3.7: py37, lint
+  3.7: py37, lint, typing
```

Comment:
Please include all the changes from the other PR.

Reply:
ups, I did "git cherry-pick tox" and thought it cherry picked the whole branch, my bad.

Done